### PR TITLE
[Tools] Removing assertion on legacy crc file creation.

### PR DESCRIPTION
**Pull request by kgills**

### CODE REVIEW

**Comment by kgills** [comment]
Refer to #185 .

File: tools/post_compute_crc.py [hunk 1]
```diff
@@ -132,10 +132,8 @@ def main():
         offset_update.create_padded_image(output_file_legacy,
                                           output_file_legacy_5000,
                                           start, pad_addr, 0x40)
-    elif start == 0 or start == 0x80000:
-        pass
     else:
```
Comment (c1728p9) [inline tools/post_compute_crc.py]: Can you remove the else case as well?

**Comment by kgills** [comment]
Removed the else condition.

**Comment by c1728p9** [comment]
Thanks

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Pull request merged**